ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -3961,6 +3961,51 @@ interface Ethernet81/2
 interface Ethernet81/10
    description isis_port_channel_member
    channel-group 110 mode active
+!
+interface Ethernet82
+   description Switchport_tap_tool
+   switchport tap native vlan 10
+   switchport tap identity 3 inner 5
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap mpls pop all
+   switchport tap encapsulation gre protocol 0x0001 strip
```

Comment:
this is rendered as

` switchport tap encapsulation gre protocol 0x1 strip` not `0x0001` (need to check the tempaltes to conclude myself)

Reply:
yes, I kept it same as the command